infer-web.py

```diff
@@ -18,6 +18,14 @@
 import gradio as gr
 import numpy as np
 import torch
+try:
+    import intel_extension_for_pytorch as ipex # pylint: disable=import-error, unused-import
+    if torch.xpu.is_available():
+        from infer.modules.ipex import ipex_init
+        ipex_init()
+        print("Using IPEX")
```

Comment:
Consider using logger.info to print a log.

Reply:
Fair